src/features/AnnouncementModal/Announcements.ts

```diff
@@ -0,0 +1,26 @@
+import { IsomerThumbsUp } from "assets"
+import { IsomerWaitingLine } from "assets/images/IsomerWaitingLine"
+import { Announcements } from "types/announcements"
+
+// TODO: Add link
+export const ANNOUNCEMENTS: Announcements[] = [
```

Comment:
Should this be in some kind of config/constants folder rather than under features/components

Reply:
actually i'm open to this also - no strong opinion but i saw the original form one keep it here so that it avoids conflation w/ actual announcement constants (eg: max ann, static isomer guide url etc)

i can change this if you want to!

Reply:
Am okay with either. Personally, I will keep it in a config file separately

Reply:
Alright, I'll resolve this!